src/NzbDrone.Core/Parser/IsoLanguages.cs

```diff
@@ -8,47 +8,56 @@ public static class IsoLanguages
-                                                               new IsoLanguage("ko", "kor", Language.Korean),
-                                                               new IsoLanguage("hu", "hun", Language.Hungarian),
-                                                               new IsoLanguage("he", "heb", Language.Hebrew),
-                                                               new IsoLanguage("cs", "ces", Language.Czech),
-                                                               new IsoLanguage("an", "any", Language.Any)
+                                                               new IsoLanguage("en", "", "eng", Language.English),
+                                                               new IsoLanguage("fr", "", "fra", Language.French),
+                                                               new IsoLanguage("es", "", "spa", Language.Spanish),
+                                                               new IsoLanguage("de", "", "deu", Language.German),
+                                                               new IsoLanguage("it", "", "ita", Language.Italian),
+                                                               new IsoLanguage("da", "", "dan", Language.Danish),
+                                                               new IsoLanguage("nl", "", "nld", Language.Dutch),
+                                                               new IsoLanguage("ja", "", "jpn", Language.Japanese),
+                                                               new IsoLanguage("is", "", "isl", Language.Icelandic),
+                                                               new IsoLanguage("zh", "", "zho", Language.Chinese),
+                                                               new IsoLanguage("ru", "", "rus", Language.Russian),
+                                                               new IsoLanguage("pl", "", "pol", Language.Polish),
+                                                               new IsoLanguage("vi", "", "vie", Language.Vietnamese),
+                                                               new IsoLanguage("sv", "", "swe", Language.Swedish),
+                                                               new IsoLanguage("no", "", "nor", Language.Norwegian),
+                                                               new IsoLanguage("nb", "", "nob", Language.Norwegian),
+                                                               new IsoLanguage("fi", "", "fin", Language.Finnish),
+                                                               new IsoLanguage("tr", "", "tur", Language.Turkish),
+                                                               new IsoLanguage("pt", "", "por", Language.Portuguese),
+                                                               new IsoLanguage("pt", "br", "por", Language.BrazilianPortuguese),
```

Comment:
I belive 3 letters code is pob (for brazilian portuguese)

Reply:
"ptb" is an option in this context?

Reply:
Do you have some backup to that? Everything I referenced says there is no iso 639-2 three digit code for Brazilian Portuguese, given it’s a dialect and not it’s own language. pob is a deprecated code for Western Pokomchí

Reply:
No, I haven't. I really don't know if using "ptb" or "pob" in this file helps brazillians to find better results. But theses codes are used in big sites (like opensubtitles, for "pob", and a lot of places (UPS, IBM, Oracle, Microsoft, https://www.google.com/search?q=%22ptb%22+language+code), for "ptb" ).
If you chose to be purist and keep this file with only iso 639-2, would by nice to create some strategy to use theses other codes too.

Reply:
Ok sounds good, I just want to make sure there is no interference as there was on the eng codes fixed here (uk also being iso for Ukraine)

Reply:
I believe that it is necessary to use the ISO 3166-1 tag in the Radarr code.

This will open the way to support all the languages of TMDb.

https://developers.themoviedb.org/3/getting-started/languages

Reply:
Yes, that is included in this PR